test/videojs-contrib-hls.test.js

```diff
@@ -42,6 +43,9 @@ QUnit.module('HLS', {
     this.clock = this.env.clock;
     this.old = {};
 
+    this.old.originalSelectPlaylist = Hls.STANDARD_PLAYLIST_SELECTOR;
+    Hls.STANDARD_PLAYLIST_SELECTOR = movingAverageBandwidthSelector();
```

Comment:
Yes, but because of the way the closure works, if we don't do this between tests, each test will be using the moving average of all the previous tests